src/export/ExportMP3.cpp

```diff
@@ -1757,15 +1758,6 @@ bool ExportMP3::CheckFileName(wxFileName & WXUNUSED(filename), int WXUNUSED(form
    return true;
 }
 
-int ExportMP3::SetNumExportChannels()
-{
-   bool mono;
-   gPrefs->Read(wxT("/FileFormats/MP3ForceMono"), &mono, 0);
```

Comment:
If you change lines like this, also take the opportunity to eliminate one more repeated string literal path with a `BoolSetting` object.